Automattic/wp-calypso
# Pricing Grid: Add new signup flow and feature flag

#### Proposed Changes

* This PR copies over the existing `plan-features-comparison` folder to a new folder, creating a new `PlanFeatures2023Grid` component.
* The new `PlanFeatures2023Grid` component, which is right now exactly the same as the current `PlanFeaturesComparison` component, is hidden behind a feature flag and will be rendered only in a new `/start/onboarding-2023-pricing-grid` signup flow. 
* Copying over to a new folder helps separate the new pricing grid work, and this PR will set the base for upcoming PRs to style the plans step to the new designs shared in pdgrnI-1Qp-p2.

#### Testing Instructions



* Go through the signup flow `/start/` and verify that the signup flow plans step works well on desktop and mobile without any console errors or issues. 
* Go through `/start/onboarding-2023-pricing-grid` and everything should work normally. There are no UI changes of any sort introduced. 

#### Pre-merge Checklist





Related to #
